en/docs/WIP/configuring-api-security-audit.md

```diff
@@ -1,82 +1,87 @@
 
-    ![]({{base_path}}/assets/img/Learn/api-token-create-button.png)
+4.  Enter a name for the token and select **API Contract Security Audit** as the token access right. 
 
-4.  Enter a name for the token and select **API Contract Security Audit** under token access right. For the audit feature to work, only the **API Contract Security Audit** permission is required.
+     When working with API Audit, you only need to select the **API Contract Security Audit** permission.
 
-    ![]({{base_path}}/assets/img/Learn/api-token-dialog-box.png)
+     [![api token dialog box]({{base_path}}/assets/img/Learn/api-token-dialog-box.png)]({{base_path}}/assets/img/Learn/api-token-dialog-box.png)
 
-5.  Click on the **Generate Token**.
+5.  Click **Generate Token**.
 
     !!! note 
-        The generated API Token can be viewed only once. Make sure to copy it and save it in a safe place for reference.
+        The generated API Token can be viewed only once. Make sure to copy it and save it in a safe place for future reference.
 
-    ![]({{base_path}}/assets/img/Learn/api-token-copy-generated.png)
+     [![API token copy generated]({{base_path}}/assets/img/Learn/api-token-copy-generated.png)]({{base_path}}/assets/img/Learn/api-token-copy-generated.png)
 
-6.  Create a collection by clicking on **API Collections** from the left navigation bar or by clicking on **+ New Collection** at the bottom of the left navigation.
+6.  Click either **API Collections** in the left navigation bar or click **+ New Collection** at the bottom of the left navigation to create a collection.
 
-    !!! note
-        A Collection in this context is a folder hosted on 42Crunch containing all the APIs that are to be audited.
+     A Collection in this context is a folder hosted on 42Crunch containing all the APIs that are to be audited.
 
-    ![]({{base_path}}/assets/img/Learn/collection-create-new.png)
+     [![create new collection]({{base_path}}/assets/img/Learn/collection-create-new.png)]({{base_path}}/assets/img/Learn/collection-create-new.png)
 
 7.  Copy the Collection ID from the URL of the browser as highlighted below and save it in a safe place for reference.
 
-    ![]({{base_path}}/assets/img/Learn/collection-copy-id.png)
+     [![copy collection id]({{base_path}}/assets/img/Learn/collection-copy-id.png)]({{base_path}}/assets/img/Learn/collection-copy-id.png)
 
-### Configure WSO2 API-M with the retrieved properties
+### Step 1.2 - Configure WSO2 API-M
 
-The API Token and Collection ID properties have to be placed inside the configuration files inside WSO2 API-M. There are two ways to do so.
+You need to add the API Token and Collection ID properties inside the configuration files in WSO2 API-M in order to configure WSO2 API-M with the retrieved properties. Use one of the following methods to carry out this process.
 
-#### Enabling the feature for all tenants
+#### Enable API security audit for all tenants
```

Comment:
We could make this "**Audit API**" since that term has been used elsewhere in the doc

Reply:
Fixed in 
https://github.com/wso2/docs-apim/pull/691/commits/38cee297b92e014c794b38f3e71091b8f9718df0